admin/build-doc

```diff
@@ -51,6 +51,7 @@ fi
 set -e
 
 cat $TOPDIR/src/osd/PG.h $TOPDIR/src/osd/PG.cc | $TOPDIR/doc/scripts/gen_state_diagram.py > $TOPDIR/doc/dev/peering_graph.generated.dot
+dot -Tsvg $TOPDIR/doc/dev/peering_graph.generated.dot -o $TOPDIR/doc/dev/peering_graph.generated.svg -Gratio=fill
```

Comment:
might want to remove `*.generated.dot` from .gitignore. and add `*.generated.svg` to `doc/.gitignore`, assuming we will have more of them in future =)